test/jdk/java/lang/management/OperatingSystemMXBean/GetSystemLoadAverage.java

```diff
@@ -54,25 +58,25 @@ public class GetSystemLoadAverage {
     // Allow some delta.
     private static double DELTA = 0.05;
 
-    public static void main(String args[]) throws Exception {
-        if (args.length > 1)  {
-            throw new IllegalArgumentException("Unexpected number of args " + args.length);
-        }
+    private static final String OS = System.getProperty("os.name");
+
+    @BeforeMethod
+    void beforeMethod() throws InterruptedException {
+        System.out.println("Wait for 5 seconds.");
+        Thread.sleep(5000);
+    }
 
-        if (args.length == 0) {
+    @Test(invocationCount = 5, timeOut = 300)
+    void testSystemLoadAvg() throws Exception {
+        if (!OS.contains("Win")) {
```

Comment:
Check /test/lib/jdk/test/lib/Platform.java, it contains "Platform.isWindows()" which could be used for this.